### ISSUE TITLE
NOT AN ISSUE BUT A DISCUSSION. Question about DLSS 3 to you since you are an expert

### ISSUE THREAD

**Issue by NoUserNameForYou** [PotatoOfDoom/CyberFSR2#67]
Hey man,

Is NVidia lying about 3 series' optical flow accelerator being "too slow" to do DLSS 3.0 frame generation? 

TV L1 Optical Flow perf of 

3080ti: 172

https://compubench.com/device.jsp?did=98267658&api=cl&os=Windows&hwtype=dGPU&D=NVIDIA+GeForce+RTX+3080+Ti 

4080: 235

https://compubench.com/device.jsp?did=110613276&api=cl&os=Windows&hwtype=dGPU&D=NVIDIA+GeForce+RTX+4080

I've seen examples of 4 series doing 40% more FPS with frame generation, I also tested some algorithms in flowframes, 720p 30fps videos can easily hit x2 speeds realtime while using a small amount of gpu resources. 

Hell, my TV from 2013 can do it (albeit has an asic in it I guess) and not to mention SVP project...

Are they lying to upsell their price gouged cards?

**Comment by InanimateCrbnRod** [comment]
I would like to know this as well because I waited in line at 3am in the freezing cold to get my 3070 and it was outdated before I beat any AAA titles on it lmao

**Comment by NoUserNameForYou** [comment]
> I would like to know this as well because I waited in line at 3am in the freezing cold to get my 3070 and it was outdated before I beat any AAA titles on it lmao

Yes. They are lying https://old.reddit.com/r/nvidia/comments/11ge42z/frame_generator_ofa_and_tensor_cores_usage/

**Comment by mirh** [comment]
https://github.com/Nukem9/dlssg-to-fsr3
